contrib/test-install-suite

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+if [ "x$PIPENV_ACTIVE" != x ]
```

Comment:
Since this is bash, you can use the `[[` test and avoid some of these ugly posix-isms

Reply:
Good point. That script should not be `#!/bin/bash`, but `#!/bin/sh` or `#!/usr/bin/python3`.

Reply:
Changed to /bin/sh in session-dbus.